mp/src/public/tier1/utlsoacontainer.h

```diff
@@ -109,6 +109,95 @@ template<class T> class CStridedConstPtr
 	}
 };
 
+class CFltx4StridedPtr
```

Comment:
The manual instantiation is unfortunate, but according to a gcc warning the __vector-specifier would get lost in the templated-code, meaning that (slow) unaligned access to the floats was used. By manually expanding CStridedPtr<fltx4> the compiler can emit the proper vectorized access instructions.
I don't know if MSVC actually just does carry over the specifiers or like the old gcc silently throws it away.